scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/Schema.scala

```diff
@@ -0,0 +1,97 @@
+package scala.meta
+package internal
+package semantic
+
+import java.net.URI
+import org.scalameta.data._
+import scala.{Seq => _}
+import scala.collection.immutable.Seq
+import scala.meta.io._
+import scala.meta.internal.semantic.{vfs => v}
+import scala.meta.internal.semantic.{schema => s}
+import scala.meta.{semantic => m}
+import scala.meta.inputs.{Input => mInput, Position => mPosition, Point => mPoint}
+
+package object schema {
+  @data class Database(entries: Seq[Attributes]) {
```

Comment:
For #777, this should read `entries: Seq[(RelativePath, Attributes)]`. Relative paths are already encoded in `v.Entry.name`, so we don't need to store them in Attributes at all.